spikeinterface/core/core_tools.py

```diff
@@ -146,6 +146,19 @@ def read_binary_recording(file, num_chan, dtype, time_axis=0, offset=0):
     return samples
 
 
+def apply_dimensions_to_positions(positions: np.ndarray, dimensions: str):
```

Comment:
We already have something similar in probeinterface.
`_2d_to_3d()` I would vote for a unique implemenation in probeinterface to avoid duplicated logic, no ?